cart/admin.py

```diff
@@ -1,12 +1,23 @@
+from typing import List
+
 from django.contrib import admin
+
 from .models import (
-    Product, OrderItem, Order, ColourVariation,
-    SizeVariation, Address, Payment, Category, StripePayment
+    Product,
+    OrderItem,
+    Order,
+    ColourVariation,
+    SizeVariation,
+    Address,
+    Payment,
+    Category,
+    StripePayment,
 )
 
 
+@admin.register(Address)
```

Comment:
we immediately register it through the decorator so as not to mix it with the rest